package.json

```diff
@@ -13,9 +15,12 @@
     "typecheck": "tsc",
     "db:push": "prisma db push --accept-data-loss",
     "db:seed": "prisma db seed --preview-feature",
-    "cypress:open": "cypress open",
-    "cypress": "NEXT_PUBLIC_ENV=test NODE_ENV=test run-p dev cypress:open",
     "test": "jest",
+    "test:db:push": "dotenv -e .env.test.local run-s db:push",
```

Comment:
Do these additional bits need to go across to the github actions workflow? Be careful here though, some, including the baseline scan need to run under production for some features to be on.

Reply:
No I don't think so because I didn't remove the commands used in our workflows and only created defaults for development and test. What do you think?

Reply:
I reckon this is alright 👍